### ISSUE TITLE
chef-server-ctl returns 0 when commands fail

### ISSUE THREAD

**Issue by sdelano** [chef/chef-server#65]
```
root@default-ubuntu-1204:~# chef-server-ctl user-create foobar foo bar boo@bar.com  testPass --filename /vagrant/foo-bar.pem
```
```
ERROR: The data in your request was invalid
Response: Field 'username' invalid
root@default-ubuntu-1204:~# echo $?
0
```

**Comment by stevendanna** [comment]
https://github.com/opscode/opscode-omnibus/pull/670

**Comment by stevendanna** [comment]
Fixed in 12.0.3
